Update the feature ownership page (#10131)
PostHog/posthog.com: contents/handbook/engineering/feature-ownership.md

|---|---|---|
| Actions | [Team Product Analytics][Team Product Analytics]  | <span class="lemon-tag gh-tag">feature/actions</span> |
| Annotations | [Team Product Analytics][Team Product Analytics]  | <span class="lemon-tag gh-tag">feature/annotations</span> |

> | API Structure | Security + core updates owned by [Team CDO][Team CDP]. Features owned by the relevant small team | <span class="lemon-tag gh-tag">feature/api-structure</span> |

Reviewer: ```suggestion
| API Structure | Security + core updates owned by [Team CDP][Team CDP]. Features owned by the relevant small team | <span class="lemon-tag gh-tag">feature/api-structure</span> |
```

> | API Structure | Security + core updates owned by [Team CDO][Team CDP]. Features owned by the relevant small team | <span class="lemon-tag gh-tag">feature/api-structure</span> |

Reviewer 2: ```suggestion
| API Structure | Shared responsibility. Features owned by the relevant small team | <span class="lemon-tag gh-tag">feature/api-structure</span> |
```

| Async migrations | [Team CDP][Team CDP]  | <span class="lemon-tag gh-tag">feature/async-migrations</span> |
| Batch exports | [Team Batch Exports](/teams/batch-exports)  | <span class="lemon-tag gh-tag">feature/batch-exports</span> |
| BI | [Team Data Warehouse](/teams/data-warehouse)  |  <span class="lemon-tag gh-tag">feature/dashboards</span> |
| Billing | [Team Growth][Team Growth]  |  <span class="lemon-tag gh-tag">feature/billing</span> |

> | Client libraries | Shared responsibility with features owned by the relevant Small Team, or try #feature-client-libraries  | <span class="lemon-tag gh-tag">feature/pipeline</span> |

Reviewer: ```suggestion
| Client libraries | Shared responsibility with features owned by the relevant Small Team, or try #feature-client-libraries. There is an engineer assigned on a rotating schedule. Check [the (private) pager duty schedule](https://posthog.pagerduty.com/schedules#P7B7NTR)  | <span class="lemon-tag gh-tag">feature/pipeline</span> |
```

Reviewer: maybe too much detail for this page 🤷

> | Client libraries | Core updates owned by [Team CDP][Team CDP]. Features owned by the relevant Small Team, or try #feature-client-libraries  | <span class="lemon-tag gh-tag">feature/pipeline</span> |

Reviewer 2: ```suggestion
| Client libraries | Shared responsibility with features owned by the relevant Small Team, or try #feature-client-libraries  | <span class="lemon-tag gh-tag">feature/pipeline</span> |
```

> | Client libraries | Core updates owned by [Team CDP][Team CDP]. Features owned by the relevant Small Team, or try #feature-client-libraries  | <span class="lemon-tag gh-tag">feature/pipeline</span> |

Reviewer: ```suggestion
| Client libraries | Core updates owned by [Team CDP][Team CDP]. Features owned by the relevant Small Team, or try #feature-client-libraries. There is an engineer assigned on a rotating schedule. Check [the (private) pager duty schedule](https://posthog.pagerduty.com/schedules#P7B7NTR)  | <span class="lemon-tag gh-tag">feature/pipeline</span> |
```

i would guess Team CDP don't want to own SDKs but interested to see what they say

> | Client libraries | Shared responsibility with features owned by the relevant Small Team, or try #feature-client-libraries. There is an engineer assigned to SDK support on a rotating schedule. Check [the (private) pager duty schedule](https://posthog.pagerduty.com/schedules#P7B7NTR)  | <span class="lemon-tag gh-tag">feature/pipeline</span> |

Reviewer 2: ```suggestion
| Client libraries and SDKs | Shared responsibility with features owned by the relevant Small Team, or try #feature-client-libraries. There is an engineer assigned to SDK support on a rotating schedule. Check [the (private) pager duty schedule](https://posthog.pagerduty.com/schedules#P7B7NTR)  | <span class="lemon-tag gh-tag">feature/pipeline</span> |
```

| Notebooks | [@daibhin][@daibhin]  |  <span class="lemon-tag gh-tag">feature/notebooks</span> |
| Onboarding | [Team Growth][Team Growth]  | <span class="lemon-tag gh-tag">feature/onboarding</span>  |
| Permissions | [@Twixes][@Twixes]  | <span class="lemon-tag gh-tag">feature/permissions</span>  |
| Persons | [Team CDP][Team CDP]  | <span class="lemon-tag gh-tag">feature/persons</span>  |
| Pipeline Transformations | [Team CDP][Team CDP] | <span class="lemon-tag gh-tag">feature/pipelines</span> |
| Pipeline Destinations | [Team CDP][Team CDP] | <span class="lemon-tag gh-tag">feature/cdp</span> |
| Platform (US + EU) | [Team Infrastructure][Team Infrastructure] | <span class="lemon-tag gh-tag">feature/platform</span>  |
| Project Home Page | [Team Product Analytics][Team Product Analytics]  | <span class="lemon-tag gh-tag">feature/home</span> |
| Property Filters | [Team Product Analytics][Team Product Analytics]  | <span class="lemon-tag gh-tag">feature/filters</span>  |
| Replay | [Team Replay][Team Replay]  |  <span class="lemon-tag gh-tag">feature/replay</span> |

> | SDKs | Shared responsibility with features owned by the relevant Small Team, or try #feature-client-libraries  | <span class="lemon-tag gh-tag">feature/pipeline</span> |

Reviewer 2: ```suggestion
```

> | SDKs | Core updates owned by [Team CDP][Team CDP]. Features owned by the relevant Small Team, or try #feature-client-libraries  | <span class="lemon-tag gh-tag">feature/pipeline</span> |

Reviewer 2: ```suggestion
| SDKs | Shared responsibility with features owned by the relevant Small Team, or try #feature-client-libraries  | <span class="lemon-tag gh-tag">feature/pipeline</span> |
```

[Team Product Analytics]: /teams/product-analytics
[Team Web Analytics]: /teams/web-analytics
[Team Replay]: /teams/replay

> [Team CDP]: /teams/pipeline

Reviewer 2: ```suggestion
[Team CDP]: /teams/cdp
```
